homeassistant/components/light/lifx.py

```diff
@@ -265,6 +269,9 @@ def async_turn_on(self, **kwargs):
     @asyncio.coroutine
     def async_turn_off(self, **kwargs):
         """Turn the device off."""
+        if not self.available:
+            return
+
```

Comment:
I'm not sure if that make more sense to use it on Export_entity on ServiceComponent...